<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QDataStream读写文件</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>10</y>
      <width>551</width>
      <height>346</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>表格数据文件</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="editTriggers">
         <set>QAbstractItemView::AllEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>666</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actTabReset"/>
   <addaction name="actOpen"/>
   <addaction name="actSave"/>
   <addaction name="separator"/>
   <addaction name="actOpenBin"/>
   <addaction name="actSaveBin"/>
   <addaction name="separator"/>
   <addaction name="actAppend"/>
   <addaction name="actInsert"/>
   <addaction name="actDelete"/>
   <addaction name="separator"/>
   <addaction name="actExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/open.bmp</normaloff>:/images/icons/open.bmp</iconset>
   </property>
   <property name="text">
    <string>打开stm文件</string>
   </property>
   <property name="toolTip">
    <string>打开预定义编码文件</string>
   </property>
  </action>
  <action name="actSave">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/save.bmp</normaloff>:/images/icons/save.bmp</iconset>
   </property>
   <property name="text">
    <string>保存stm文件</string>
   </property>
   <property name="toolTip">
    <string>保存为预定义编码文件</string>
   </property>
  </action>
  <action name="actAppend">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/append.bmp</normaloff>:/images/icons/append.bmp</iconset>
   </property>
   <property name="text">
    <string>添加行</string>
   </property>
   <property name="toolTip">
    <string>添加一行</string>
   </property>
  </action>
  <action name="actInsert">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/306.bmp</normaloff>:/images/icons/306.bmp</iconset>
   </property>
   <property name="text">
    <string>插入行</string>
   </property>
   <property name="toolTip">
    <string>插入一行</string>
   </property>
  </action>
  <action name="actDelete">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/delete.bmp</normaloff>:/images/icons/delete.bmp</iconset>
   </property>
   <property name="text">
    <string>删除行</string>
   </property>
   <property name="toolTip">
    <string>删除当前行</string>
   </property>
  </action>
  <action name="actExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/exit.bmp</normaloff>:/images/icons/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actModelData">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/import1.bmp</normaloff>:/images/icons/import1.bmp</iconset>
   </property>
   <property name="text">
    <string>模型数据预览</string>
   </property>
   <property name="toolTip">
    <string>模型数据显示到文本框里</string>
   </property>
  </action>
  <action name="actAlignLeft">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/508.bmp</normaloff>:/images/icons/508.bmp</iconset>
   </property>
   <property name="text">
    <string>居左</string>
   </property>
   <property name="toolTip">
    <string>文字左对齐</string>
   </property>
  </action>
  <action name="actAlignCenter">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/510.bmp</normaloff>:/images/icons/510.bmp</iconset>
   </property>
   <property name="text">
    <string>居中</string>
   </property>
   <property name="toolTip">
    <string>文字居中</string>
   </property>
  </action>
  <action name="actAlignRight">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/512.bmp</normaloff>:/images/icons/512.bmp</iconset>
   </property>
   <property name="text">
    <string>居右</string>
   </property>
   <property name="toolTip">
    <string>文字右对齐</string>
   </property>
  </action>
  <action name="actFontBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/500.bmp</normaloff>:/images/icons/500.bmp</iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
   <property name="toolTip">
    <string>粗体字体</string>
   </property>
  </action>
  <action name="actTabReset">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/100.bmp</normaloff>:/images/icons/100.bmp</iconset>
   </property>
   <property name="text">
    <string>表格复位</string>
   </property>
   <property name="toolTip">
    <string>表格复位</string>
   </property>
  </action>
  <action name="actOpenBin">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/126.bmp</normaloff>:/images/icons/126.bmp</iconset>
   </property>
   <property name="text">
    <string>打开dat文件</string>
   </property>
   <property name="toolTip">
    <string>打开标准编码文件</string>
   </property>
  </action>
  <action name="actSaveBin">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/icons/066.GIF</normaloff>:/images/icons/066.GIF</iconset>
   </property>
   <property name="text">
    <string>保存dat文件</string>
   </property>
   <property name="toolTip">
    <string>保存为标准编码文件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
